src/Interro.java

```diff
@@ -89,39 +97,22 @@ public void paintComponent(Graphics g){
     class Game extends JPanel{
         int alive;
         int time;
-        Grid[][] grid;
+        int gLength, cLength;
+        int[][] rocks;
         public Game(){
-            grid = new Grid[10][10];
+            setBackground(Color.decode("#329da8"));
+            gLength = cLength = 10;
+            rocks = new int[300][2];
+            for(int i = 0; i < rocks.length; i++) for(int j = 0; j < rocks[0].length; j++) rocks[i][j] = (int)(Math.random()*965+15);
```

Comment:
Put the second `for` on a separate line. The same for the body of the `for` statement. Also, use braces for the bodies of the `for` statements (always use braces for bodies of `for`, `if`, etc).

`965` and `15` are called "magic numbers". Turning them into named constants would make the code more understandable.